docs/E-Land_Requirements.rst

```diff
@@ -445,11 +432,16 @@ NF-MVS-02 - MVS post-processing tools for LES optimization model output/results
 The post-processing of results ensures that important KPI can be provided for the energy system optimization.
 There are three output formats of the MVS:
 
-* For the end-user of the standalone application, an automatic report is generated that makes scenario evaluation easy
-* For a developer of the standalone application, the results are also provided as excel files and pngs.
+* For the end-user of the standalone application, an :ref:`automatic report <output_report>` is generated that makes scenario evaluation easy
+* For a developer of the standalone application, the results are also provided as excel files and pngs
 * For the EPA, the results are provided in a json format to be displayed interactively in their environment
 
-:ToDo: Improving the outputs is a continuing task.
+:Notes:
+
+Improving the outputs is a continuing task. Following improvements can be considered in the future:
+
+* Move all KPI connected to the individual energy vectors into a seperate table and display in the report
+* Add-on requested by end-users: Cash flow projections
 
```

Comment:
Display of units should be improved, as well, but I don't know whether it's suitable to state this here.

Reply:
I would not include it here, it is more of a bug/patch then a feature.